Add new issues templates (#1503)
freeCodeCamp/devdocs: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve Devdocs
title: ''

> labels: ''

Reviewer: Can we assign appropriate labels automatically?

assignees: ''
---

<!--
If possible fill each section
-->

# Bug report

<!--
Verify this steps before writing a new issue

>  - Update to the most recent master release; the bug may already be fixed.

Reviewer: This item is not relevant when people are using devdocs.io?

 - Search for existing issues; it's possible someone has already encountered this bug.
-->

## OS information

<!--

> Write what operative system and web browser you use

Reviewer 2: ```suggestion
What operating system and browser version are you using?
```

-->

## Steps to reproduce

<!--

> Write the steps to reproduce this bug or write a description about when and how is
> this happening

Reviewer 2: ```suggestion
Write the steps to reproduce this bug or write a description about when and how you
encountered it
```
